wger/manager/models.py

```diff
@@ -69,6 +69,7 @@ class Meta:
                     "example 'Focus on back' or 'Week 1 of program xy'."))
     user = models.ForeignKey(User,
                              verbose_name=_('User'), on_delete=models.CASCADE)
+    imported = models.BooleanField(blank=True, default=False)
```

Comment:
Since you have set the default to False, is it necessary to set blank to True?

Reply:
Nice catch. I don't think it's necessary. I'll remove it.